src/stories/3-forms/EnhancedInputUpload.stories.tsx

```diff
@@ -13,7 +13,9 @@ export default {
 const Template: ComponentStory<any> = args => {
     const { wrapperProps, ...otherProps } = args;
 
-    const handleFiles = files => console.log(files);
+    const handleFiles = files => setTimeout(() => {
+        console.log(files);
```

Comment:
is this console.log needed?

Reply:
That's the storybook component, doesn´t affect the component behaviour on the app and makes it possible to see the loading state on the storybook